### ISSUE TITLE
Normalization variance calculation backprop

### ISSUE THREAD

**Issue by abhiskk** [darkstar112358/ladder#2]
While calculating variance for the modified batch normalization the variance which is calculated is done after getting detached from the backprop which breaks the backprop.
